mona_sdk/client.py

```diff
@@ -114,15 +115,20 @@
     "SAMPLING_FACTORS_MAX_AGE_SECONDS", 300
 )
 
-SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service."
+SERVICE_ERROR_MESSAGE = "Could not get server response for the wanted service"
 UPLOAD_CONFIG_ERROR_MESSAGE = (
-    "Could not upload the new configuration, please check it is valid."
+    "Could not upload the new configuration, please check it is valid. "
 )
-APP_SERVER_CONNECTION_ERROR_MESSAGE = "Cannot connect to app-server."
+RETRIEVE_CONFIG_HISTORY = "Retrieve history is empty"
+GET_AGGREGATED_STATS_OF_SPECIFIC_SEGMENTATION_ERROR = (
+    "Could not get aggregates state of a specific segmentation"
+)
+SAMPLING_FACTORS_ERROR = "sampling config names and their sampling map list are empty"
```

Comment:
the message is about the sampling lists being empty, not a general error, so its best if the name of this env will reflect that (SAMPLING_FACTORS_EMPTY_MAPPING_ERROR)